src/pages/Home.jsx

```diff
@@ -4,10 +4,17 @@ import { apiPath } from "../api";
 import Navbar from "./Navbar";
 import "../styles/home.css";
 import Testimonials from "../component/Testimonials";
+import PriceCard from "../component/PriceCard";
 import About from "../component/About";
 
 const Home = () => {
   const [teamsDatabase, setTeamsDatabase] = useState([]);
+  const current_theme = localStorage.getItem('current_theme');  // This will store the default theme of web browser
+  const [theme, setTheme] = useState(current_theme ? current_theme : 'light'); // Created constant for dark mode

```

Comment:
Comments are usually not needed for small and simple parts that are self-explanatory. 
(These days it can also signal that our friend robot was used, as it likes to comment explicitly 😅 )

Reply:
Removed comments.